walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,210 @@
 // limitations under the License.
 
 package com.google.sps;
+import com.google.sps.*;
+import java.util.*;
+public final class FindMeetingQuery {
 
-import java.util.Collection;
+  /* Given all events, return a HashMap that contains the schedule of mandatory attandees and optional attandees 
+  param: @events: all events containing time and attandees list 
+         @request: Meeting requests containing attandees list
+  return: @allSchedules a hashmap with 2 key: 'mandatory' and 'optional', the values are another HashMap whose key is attandee and 
+          value is the schedule of that attandee
+  */
+  public static HashMap<String, HashMap<String, ArrayList<TimeRange>>> getSchedules(Collection<Event> events, MeetingRequest request) {
+    Collection<String> mandatoryAttandeeSet = request.getAttendees();
```

Comment:
Nit: Spelling should be `mandatoryAttendeeSet` and `optionalAttendeeSet`. I think this spelling is used through this file, would be good to fix.

Reply:
Would be good to address this nit, the spelling is fixed in some places but not others.